### DISCUSSION TITLE
Question about the two connections in the Vectorwar example

### DISCUSSION THREAD

**Discussion by digiholic** [nykwil/UnityGGPO#12] [Q&A]
The connections panel in the VectorWar example has two input boxes labeled "Connections". One is `inp` and the other is `txt` in the GameManager. They are used to generate the connections in `GetConnections()`. Both have a checkbox for spectator.

When running two copies on my local machine, I can't get a connection until both values are the same, and the spectator boxes seem to do nothing at all. My question is, what are these two boxes? Do you need both your own as well as your opponent's IP to connect? Or are both of these connections expected to be different ports on the opponent's machine?

**Comment by nykwil** [comment]
The way the UDP connection works right now is you need the IP on both
clients. They will have to be different ports if they are on the same
machine.
